src/modules/fancyzones/lib/FancyZones.cpp

```diff
@@ -218,8 +215,9 @@ struct FancyZones : public winrt::implements<FancyZones, IFancyZones, IFancyZone
     OnThreadExecutor m_dpiUnawareThread;
     OnThreadExecutor m_virtualDesktopTrackerThread;
 
-    static UINT WM_PRIV_VDCHANGED; // Message to get back on to the UI thread when virtual desktop changes
-    static UINT WM_PRIV_VDINIT; // Message to get back to the UI thread when FancyZones are initialized
+    static UINT WM_PRIV_VD_INIT; // Message to get back to the UI thread when FancyZones are initialized

```

Comment:
typo: is initialized
and below : to get back to (not on to)

Reply:
all three have typos :)

Reply:
Done.